doc/lsst.drp.tasks/tasks/lsst.drp.tasks.reprocess_visit_image.ReprocessVisitImageTask.rst

```diff
@@ -0,0 +1,71 @@
+.. lsst-task-topic:: lsst.drp.tasks.reprocess_visit_image.ReprocessVisitImageTask
+
+#######################
+ReprocessVisitImageTask
+#######################
+
+``ReprocessVisitImageTask`` applies calibration-model objects produced by upstream tasks to single-visit images, and performs the final round of detection, deblending, and measurement that will populate the Source table.
+
+.. _lsst.drp.tasks.reprocess_visit_image.ReprocessVisitImageTask-summary:
+
+Processing summary
+==================
+
+``ReprocessVisitImageTask`` reads in the one or two post-ISR single-exposure images that correspond to a single ``{visit, detector}`` combination, combines them into a single image (via :lsst-task:`~lsst.pipe.base.snapCombine.SnapCombineTask`).
+It then applies calibration-model objects produced upstream.
+These include:
+
+- the point-spread function model model (`lsst.afw.detection.Psf`)
+- aperture corrections (`lsst.afw.image.ApCorrMap`)
+- the astrometric solution (`lsst.afw.geom.SkyWcs`)
+- the photometric solution (`lsst.afw.image.PhotoCalib`)
+- a background model (`lsst.afw.math.BackgroundList`)
+
+``ReprocessVisitImageTask`` then detects new sources (:lsst-task:`~lsst.meas.algorithms.detection.SourceDetectionTask`), deblends them (:lsst-task:`~lsst.meas.deblender.SourceDeblendTask`), measures them (:lsst-task:`~lsst.meas.base.SingleFrameMeasurementTask`), and applies the aperture corrections loaded previously (:lsst-task:`~lsst.meas.algorithms.ApplyApCorrTaskTask`).
+Additional subtasks or configurations of :lsst-task:`~lsst.meas.base.SingleFrameMeasurementTask` may be used to ensure the calibration models are correctly reflected in all measurements, but these should be considered implementation details, despite being visible (and configurable) as regular subtasks.
+
+The tasks outputs are an `lsst.afw.image.ExposureF` that aggregates the calibration models and the `~lsst.afw.table.SourceCatalog` of single-visit measurements.
```

Comment:
I don't understand this sentence.  An Exposure that aggregates the calibration models?

Reply:
How about:

```The task's outputs are an `lsst.afw.image.ExposureF` that has been reprocessed and calibrated with the input calibration models and the `~lsst.afw.table.SourceCatalog` of single-visit measurements on that exposure.
```